docs/content/SRS/SRS.md

```diff
@@ -157,7 +183,54 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
+
+FR-6. The system shall allow users to upload custom system prompts (e.g., "Write like Hemingway" or "Adopt a motivational tone") to influence the overall behavior of the model.
+
+FR-7. The system shall allow users to navigate through their text non-linearly by providing a visual map or tree view that displays key points, topics, or sections.
+
+FR-8. The system shall provide explanations and recommendations for adjusting each hyperparameter, allowing users to better understand how changes will affect the model's output.
+
+FR-9. For each generated text suggestion, the system shall present multiple alternatives, which users can choose from, modify, or combine to form the final text.
+
+FR-10. Users shall be able to click on a specific part of the text and select from a dropdown of alternative phrasings or sentence structures generated by the model.
+
+FR-11. The system shall offer version control features, enabling users to revert to previous versions of the document or compare different drafts side-by-side.
+
+FR-12. The system shall support an offline mode where users can continue to write and interact with the editor without internet access, using pre-downloaded or locally hosted language models for text generation.
+
+FR-13. The system shall automatically sync user data and documents when internet access is restored, ensuring that no work is lost during offline writing sessions.
+
+FR-14. Users shall be able to set specific writing goals (e.g., word count, tone consistency, argument development) which the system will track and offer suggestions to help them meet those goals.
+
+FR-15. The system shall provide progress tracking and feedback on goals, such as word count meters, tone analysis, and style consistency checks, helping users to stay aligned with their original intentions.
+
+FR-16. The system shall support text input and generation in multiple languages, enabling users to switch between languages within the same document or work in multilingual environments.
+
+FR-17. The system shall offer translation features, allowing users to translate model-generated text between languages, while maintaining the original meaning and tone as closely as possible.
+
+FR-18. The system shall include standard text editing tools such as bold, italic, underline, strikethrough, and highlighting, enabling users to format their text as they write.
+
+FR-19. The system shall allow users to track changes, providing an option to view a history of edits and revert to previous versions of their text.
+
+FR-20. The system shall offer a grammar and style checker that runs in the background, highlighting potential issues and offering suggestions for improvement without interrupting the writing flow.
+
+FR-21. The system shall allow users to categorize and tag different sections of their text (e.g., introduction, argument, conclusion), enabling better organization and navigation within large documents.
```

Comment:
ye we can follow obsidian behaviour for frontmatter support no biggie, as well as backlinks and TOC support. But also should include with FR-18 as accessibility support?

What do you think?